js/values/AssessmentResult.js

```diff
@@ -29,14 +29,15 @@ AssessmentResult.prototype.getScore = function() {
 
 /**
  * Set the score for the assessment.
- * @param {number|string} score The score to be used for the score property
+ * @param {number} score The score to be used for the score property
  * @returns {void}
  */
 AssessmentResult.prototype.setScore = function( score ) {
-	if ( !isNumber( score ) && !isString( score ) ) {
+	this.isScoreSet = true;
```

Comment:
changed this to 
if ( isNumber( score ) ) {
    this.score = score;
    this.isScoreSet = true;
}